pocs/tests/conftest.py

```diff
@@ -2,39 +2,70 @@
 import pytest
 
 from pocs.utils.config import load_config
-from pocs.utils.data import download_all_files
 from pocs.utils.database import PanMongo
 
 
 def pytest_addoption(parser):
-    parser.addoption(
-        "--hardware-test",
-        action="store_true",
-        default=False,
-        help="Test with hardware attached")
-    parser.addoption(
-        "--camera",
-        action="store_true",
-        default=False,
-        help="If a real camera attached")
-    parser.addoption("--mount", action="store_true", default=False, help="If a real mount attached")
-    parser.addoption(
-        "--weather",
-        action="store_true",
-        default=False,
-        help="If a real weather station attached")
+    parser.addoption("--hardware-test", action="store_true", default=False,
```

Comment:
"Test with all hardware attached"

But, this strikes me as problematic for multiple reasons: few will have domes, many will have some hardware available but not all, and testing with the cameras doesn't risk exposing them to water, but testing with the mount does.

Perhaps a variant would be the opposite of the simulator list: --with-hardware=camera,weather

Reply:
Good point plus that gives the same sort of calling structure as the simulator. I'll get this added in to this PR, which will require some more work so might not get to today.

Reply:
I've got this change in and will be pushing soon, but another question might be whether or not we should skip all non-hardware related tests when testing hardware. If I am interested in testing the camera I don't necessarily want to wait for the `test_pocs.py` software tests to run.

Probably can just leave it for now but might require some thinking about.